test/e2e/framework/framework.go

```diff
@@ -247,6 +247,14 @@ func (f *Framework) CreateGameServerAndWaitUntilReady(t *testing.T, ns string, g
 	return readyGs, nil
 }
 
+// CreateGameServer Creates a GameServer.
```

Comment:
I would either factor this with the above so that `CreateGameServerAnd` calls this function, or I would just inline this in the test you wrote. Frankly the latter seems simple enough, since this falls in that "just a wrapper" category that Go tends to avoid.